database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/PostDetailView.swift

```diff
@@ -57,6 +57,8 @@ struct PostDetailView: View {
           .background(RoundedRectangle(cornerRadius: 10).fill(Color(.systemGray5)))
         Button(action: {
           post.didTapSendButton(commentField: comment)
+          comment = ""
+          hideKeyboard()
```

Comment:
Does this also require an `#if canImport(UIKit)`?

Reply:
yes adding it would make sense.